packages/core-data/src/entities.js

```diff
@@ -20,8 +20,16 @@ export const DEFAULT_ENTITY_KEY = 'id';
 const POST_RAW_ATTRIBUTES = [ 'title', 'excerpt', 'content' ];
 
 // A hardcoded list of post types that support revisions.
+// Reflects post types in Core's src/wp-includes/post.php.
 // @TODO: Ideally this should be fetched from the  `/types` REST API's view context.
-const POST_TYPES_WITH_REVISIONS_SUPPORT = [ 'post', 'page' ];
+const POST_TYPE_ENTITIES_WITH_REVISIONS_SUPPORT = [
+	'post',
+	'page',
+	'wp_block',
+	'wp_navigation',
+	'wp_template',
+	'wp_template_part',
+];
```

Comment:
The current thinking is to extend the `view` context's response with support information.

Reply:
Just double-checking what you mean here (sorry if I'm missing context — so to speak!): that we have a hard-coded list of types here because calling `'/wp/v2/types?context=view'` doesn't include a `supports` array in the response for each post type?

That makes sense to me, and from looking up `wp-includes/post.php` the list here corresponds with types that all opt-in to `revisions` under `supports` 👍